docs/index.asciidoc

```diff
@@ -167,6 +167,8 @@ The `interval` option specifies the polling interval in seconds.
 * Value type is <<number,number>>
 * Default value is `30`
 
+The interval creates a waiting time after all of the hosts have been polled.
```

Comment:
maybe "This is the waiting time after all of the hosts have been polled"

Reply:
I can see why this is an important distinction to call out, and you have a good start here. It would be good to clearly state what happens at the end of the interval. My guess is that polling starts over, and so I wrote something to that effect.  Please correct me if I'm wrong. :-) 

What about something like: 
>This value sets the amount of time to wait after all hosts have been polled and before the polling starts again. 
>That is, the plugin polls all hosts, it waits for the specified interval, and then it polls all hosts again.